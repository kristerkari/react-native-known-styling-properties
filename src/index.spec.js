import { allProps, allRN2CSSProps } from "./index";

it("should have a flat array of all known React Native props", () => {
  expect(allProps).toMatchSnapshot();
});

it("should have a flat array of all known css-to-react-native props", () => {
  expect(allRN2CSSProps).toMatchSnapshot();
});
