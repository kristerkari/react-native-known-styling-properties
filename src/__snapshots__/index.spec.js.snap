// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should have a flat array of all known React Native props 1`] = `
Array [
  "alignItems",
  "alignSelf",
  "flex",
  "flexDirection",
  "flexWrap",
  "justifyContent",
  "borderWidth",
  "borderBottomWidth",
  "borderLeftWidth",
  "borderRightWidth",
  "borderTopWidth",
  "borderRadius",
  "borderBottomLeftRadius",
  "borderBottomRightRadius",
  "borderTopLeftRadius",
  "borderTopRightRadius",
  "borderColor",
  "borderLeftColor",
  "borderRightColor",
  "borderTopColor",
  "borderBottomColor",
  "borderStyle",
  "position",
  "bottom",
  "left",
  "right",
  "top",
  "zIndex",
  "height",
  "minHeight",
  "maxHeight",
  "width",
  "minWidth",
  "maxWidth",
  "overflow",
  "margin",
  "marginBottom",
  "marginLeft",
  "marginRight",
  "marginTop",
  "padding",
  "paddingBottom",
  "paddingLeft",
  "paddingRight",
  "paddingTop",
  "transform",
  "backfaceVisibility",
  "background",
  "backgroundColor",
  "color",
  "opacity",
  "fontFamily",
  "fontSize",
  "fontStyle",
  "fontWeight",
  "fontVariant",
  "lineHeight",
  "textAlign",
  "textDecorationLine",
  "textDecorationColor",
  "textDecorationStyle",
  "letterSpacing",
  "elevation",
  "textAlignVertical",
  "textShadowColor",
  "textShadowOffset",
  "textShadowRadius",
  "shadowColor",
  "shadowOffset",
  "shadowOpacity",
  "shadowRadius",
  "marginHorizontal",
  "marginVertical",
  "paddingHorizontal",
  "paddingVertical",
  "decomposedMatrix",
  "transformMatrix",
  "resizeMode",
  "tintColor",
  "overlayColor",
  "writingDirection",
]
`;

exports[`should have a flat array of all known css-to-react-native props 1`] = `
Array [
  "alignItems",
  "alignSelf",
  "flex",
  "flexDirection",
  "flexWrap",
  "justifyContent",
  "borderWidth",
  "borderBottomWidth",
  "borderLeftWidth",
  "borderRightWidth",
  "borderTopWidth",
  "borderRadius",
  "borderBottomLeftRadius",
  "borderBottomRightRadius",
  "borderTopLeftRadius",
  "borderTopRightRadius",
  "borderColor",
  "borderLeftColor",
  "borderRightColor",
  "borderTopColor",
  "borderBottomColor",
  "borderStyle",
  "position",
  "bottom",
  "left",
  "right",
  "top",
  "zIndex",
  "height",
  "minHeight",
  "maxHeight",
  "width",
  "minWidth",
  "maxWidth",
  "overflow",
  "margin",
  "marginBottom",
  "marginLeft",
  "marginRight",
  "marginTop",
  "padding",
  "paddingBottom",
  "paddingLeft",
  "paddingRight",
  "paddingTop",
  "transform",
  "backfaceVisibility",
  "background",
  "backgroundColor",
  "color",
  "opacity",
  "fontFamily",
  "fontSize",
  "fontStyle",
  "fontWeight",
  "fontVariant",
  "lineHeight",
  "textAlign",
  "textDecorationLine",
  "textDecorationColor",
  "textDecorationStyle",
  "letterSpacing",
  "elevation",
  "textAlignVertical",
  "textShadowColor",
  "textShadowOffset",
  "textShadowRadius",
  "shadowColor",
  "shadowOffset",
  "shadowOpacity",
  "shadowRadius",
  "marginHorizontal",
  "marginVertical",
  "paddingHorizontal",
  "paddingVertical",
  "decomposedMatrix",
  "transformMatrix",
  "resizeMode",
  "tintColor",
  "overlayColor",
  "writingDirection",
  "border",
  "boxShadow",
  "flexFlow",
  "font",
  "textDecoration",
  "textShadow",
]
`;
